### DISCUSSION TITLE
How to get rid of `was created with unknown prop`

### DISCUSSION THREAD

**Discussion by babakfp** [pngwn/MDsveX#510] [Q&A]
Hi

How can I get rid of this warning? I have a layout file and used it in the mdsvex layout option. I used the layout to customize the markdown image element component. So, I need the feature but I also don't want to see the warning.

![Screenshot 2023-04-30 070632](https://user-images.githubusercontent.com/44144724/235357417-1606a8a8-d593-411d-981d-1ddb1b88d1ed.png)

![Screenshot 2023-05-24 181344](https://github.com/pngwn/MDsveX/assets/44144724/4b84285b-447b-4dd7-bea7-1bf57796dba1)
![Screenshot 2023-05-24 181416](https://github.com/pngwn/MDsveX/assets/44144724/7fbb44b6-9097-4052-8633-38fb921b0a98)

Thanks

**Comment by simonsarris** [comment]
yeah it feels weird to keep these warnings around

**Comment by pngwn** [comment]
There isn't a huge amount I can do about this in mdsvex, the compiler generates these messages and the bundler displays them. You can pass a custom onwarn handler to suppress them though. 

https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/config.md#onwarn
